app/controllers/blobs_controller.rb

```diff
@@ -11,9 +11,29 @@ def create
     if File.write("received_files/" + filename, JSON.pretty_generate(json))
       json["document_id"] = uuid
       json["document_filename"] = filename
-      render json: json, status: 201
+      blob = Blob.new(
+        document_id: uuid,
+        document_filename: filename,
+        data: json.to_json
+      )
+      if blob.save
+        render json: json, status: 201
```

Comment:
I _think_ here you want to return blob and not json.  If you return `blob` then you would tell the remote system what the ID is of the record so they can look it up in the show action.  If you return `json` then you are just returning the formatted json data.  I think.

Reply:
I thought that returning JSON instead of a blob is better because it provides ease of use, multi-platform compatibility, efficient data transfer, flexibility, and scalability.